int32 AgentIdx
string vehicleType
int32 sveaId